### ISSUE TITLE
Events are not being named correctly.

### ISSUE THREAD

**Issue by mikebronner** [GeneaLabs/laravel-mixpanel#35]
Recently tracked events started being named like `{u'1': {u'Status': u'Login Attempt Failed'}, u'0': u'Session', u'time': 1497541309}`. /cc @richardbenson

**Comment by mikebronner** [comment]
Fixed by 254b56c4

**Comment by mikebronner** [comment]
@richardbenson give it a shot now.

**Comment by richardbenson** [comment]
@mikebronner 

```
FatalThrowableError in Guard.php line 273:
Type error: Argument 1 passed to Illuminate\Auth\Guard::validate() must be of the type array, null given, called in /home/vagrant/uvol-api/source/vendor/genealabs/laravel-mixpanel/src/Listeners/LaravelMixpanelEventHandler.php on line 28
```

I'm fiddling around in there to try and get the right order of things, will update if I crack it

**Comment by mikebronner** [comment]
Figures! :) Ha, made the fix in the 5.4 test platform, and not in 5.1. Testing there now, should have a fix shortly. Sorry about that.

**Comment by mikebronner** [comment]
Pushed release 0.6.11 which should alleviate the issue. Tested in both 5.1 and 5.4. Please note the release notes for some minor tracking changes made.
